tests/behat/features/reauthenticate.feature

```diff
@@ -14,24 +14,26 @@ Feature: Reauthenticate
   Scenario: Reauthenticate with correct login
     When I press the "Add Member" button
       And I switch to the "login-dialog-iframe" iframe
-    Then I should see "Your session has timed out due to inactivity" in the ".cms-security__container" element
+    Then I should see "You must be logged in" in the ".cms-security__container" element
     When I fill in "Password" with "Secret!123"
       And I press the "Let me back in" button
       And I am not in an iframe
       And I go to "/admin/security"
-      When I press the "Add Member" button
+    When I press the "Add Member" button
+      And I wait for 5 seconds until I see the "#Form_ItemEditForm_action_doSave" element
```

Comment:
This stuff isn't ideal, we have generic "after" steps which check for the presence of a loading indicator (or pending ajax request), and delay processing the next step until then. Maybe track that down? I'm happy merging as-is to unblock further work, but these "and I wait" steps are noise that should already be accounted for elsewhere.

Reply:
To be fair this one is a polling step. So `And I wait x until` is just "up to" x seconds. The actual step might only be a few milliseconds of waiting.

Reply:
Oh that makes a lot of sense, the class name has changed: https://github.com/silverstripe/silverstripe-admin/pull/765/files#diff-19f74f986482f632702ec13c64f6b019R1060
It should look for `.cms-loading-container` now. It'd be good to do a follow up on this.

Reply:
It also may mean we can revert this PR. Probably better not to change things that might not be broken if we can just wait for loading indicators correctly.

Reply:
here we go: https://github.com/silverstripe/silverstripe-framework/pull/8663

this particular scenarios are marked as `@modal` though, so we'll have to keep waiting manually, or rewrite some other stuff in `BasicContext` in `silverstripe-behat-extension`. With just removing `@modal` it's not working at this point. (not sure why exactly at this point)